sklearn/base.py

```diff
@@ -70,6 +70,15 @@ def clone(estimator, *, safe=True):
     results. Otherwise, *statistical clone* is returned: the clone might
     return different results from the original estimator. More details can be
     found in :ref:`randomness`.
+
+    Examples
+    --------
+    >>> from sklearn.base import clone
+    >>> from sklearn.svm import SVC
+    >>> classifier = SVC()
+    >>> classifier_copy = clone(classifier)
+    >>> classifier is classifier_copy # Check if classifier_copy is a deep copy.
```

Comment:
This actually only indicate that this is a copy. For a deep copy we would need to check that every internal attribute are also different.

Let's remove the comment and only show that the `is` is returning `False`.